src/page-content/content-mfa.tsx

```diff
@@ -0,0 +1,191 @@
+            src={CLIExample}
+        />
+    )
+}
+export const featureAutomation: FeatureContentProps = {
+    id: 'mfa.featurecontent.authentication',
+    overline: <>Multi-factor Authentication</>,
+    title: <>Secure Verification for Everyone</>,
+    description: (
+        <>
+            Keep your users protected at all times. Add Ory Multi-factor Authentication to your projects today.
+        </>
+    ),
+    button: {
+        ctaTitle: 'Read more',
+        to: '/docs/guides/mfa/overview',
+        iconRight: ArrowRight
+    },
+    content: (
+        <img className="responsive" loading="lazy" alt="foo" src={Automation}/>
+    )
+}
+export const usageList: FeaturelistProps = {
+    id: 'mfa.twoMethods',
+    slant: true,
+    background: 'dark',
+    features: [
+        {
+            title: 'MFA and two-step verification',
+            description: <>Two methods to do it with Ory.</>,
+            button: (
+                <Button
+                    style={'filled'}
+                    to={'/docs/guides/mfa/overview'}
+                    iconRight={ArrowRight}
+                >
+                    Try Now
+                </Button>
+            ),
+            features: [
+                {
+                    title: 'One-time password (OTP)',
+                    description: (
+                        <>
+                            {' '}
+                            Allow your users to pair with any password manager app such as Google Authenticator or
+                            OnePassword on their phone to generate a secure, temporary one-time password.
+                        </>
+                    ),
+                    icon: Keyhole,
+                },
+                {
+                    title: 'WebAuthn',
+                    description: (
+                        <>
+                            {' '}
+                            Authenticate via browser and let the user verify their identity via USB, NFC, Bluetooth, or
+                            any other biometric authentication standards.
+                        </>
+                    ),
+                    icon: Browser,
+                },
+            ]
+        }
+    ]
+}
+export const featureList: FeaturelistProps = {
+    id: 'mfa.featurelist',
+    slant: true,
+    background: 'themed',
+    features: [
+        {
+            title: 'Features of Ory Multi-factor Authentication',
+            description: <></>,
+            button: (
+                <Button
+                    style={'filled'}
+                    to={'/docs/guides/mfa/overview'}
+                    iconRight={ArrowRight}
+                >
+                    Try Now
+                </Button>
+            ),
+            features: [
+                {
+                    title: 'Configure Anywhere ',
+                    description: (
+                        <>
+                            Configure MFA using Ory CLI or even self-hosted projects.
```

Comment:
```suggestion
                            Configure MFA using the Ory CLI.
```

Should we explicitly mention self-hosting in the Cloud product pages?

Reply:
I think the motive behind mentioning it to increase signups and also to pave way for conversions. I hope that makes sense, since CLI has its own landing page.

Reply:
That makes sense but why mention self-hosted projects? The landing page is for the managed Ory Cloud right?